# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-08-14 15:15+0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: api/settings.py:241
msgid "Estonian"
msgstr "Eesti"

#: api/utilities/elastic.py:20
msgid "Could not find specified data from Elasticsearch!"
msgstr "Andmeid ei leitud!"

#: api/utilities/elastic.py:21
msgid "Error executing Elasticsearch query! Bad query?"
msgstr "Viga Elasticu päringuga. Võtke ühendust administraatoriga!"

#: api/utilities/elastic.py:23
msgid "Connection to Elasticsearch took too long, please try again later!"
msgstr "Ühendus võttis liiga kaua aega, palun proovige uuesti!"

#: api/utilities/elastic.py:25
msgid "Could not authenticate with Elasticsearch!"
msgstr "Ei saanud autentida Elasticuga! Võtke ühendust administraatoriga!"

#: api/utilities/elastic.py:26
msgid "Unexpected error from Elasticsearch!"
msgstr "Ootamatu error Elasticust! Võtke ühendust administraatoriga!"

#: api/utilities/elastic.py:28
msgid ""
"Could not connect to Elasticsearch, is the location properly configured?"
msgstr "Ei saanud ühendust Elasticuga! Võtke ühendust administraatoriga!"

#: api/utilities/gpt.py:186
msgid "No OpenAI API key given, can't query API!"
msgstr "ChatGPT ühendus on seadistamata! Võtke ühendust administraatoriga!"

#: api/utilities/gpt.py:201
msgid "Couldn't authenticate with OpenAI API!"
msgstr "Autentimine ChatGPT'ga ebaõnnestus! Võtke ühendust administraatoriga!"

#: api/utilities/gpt.py:206
msgid "OpenAI request is not correct!"
msgstr "Viga ChatGPT päringuga! Võtke ühendust administraatoriga!"

#: api/utilities/gpt.py:215
msgid "Requested resource was not found! Is the right model configured?"
msgstr ""
"Palutud ChatGPT mudel ei ole saadaval! Võtke ühendust administraatoriga!"

#: api/utilities/gpt.py:220
msgid "Requested resource was denied!"
msgstr "Ligipääs keelatud! Võtke ühendust administraatoriga!"

#: api/utilities/gpt.py:237 api/utilities/gpt.py:242
msgid "Couldn't connect to the OpenAI API!"
msgstr "Ei saanud ühendust ChatGPT'ga!"

#: api/utilities/serializers.py:17 api/utilities/serializers.py:24
msgid "The given value contains forbidden characters."
msgstr "Sisaldab keelatud tähemärke!"

#: core/mixins.py:124
msgid "Task toke too much time!"
msgstr ""
"Töö ajalimiit ületatud, palun proovige uuesti! Korduva probleemi puhul palun "
"teavitage administraatorit!"

#: core/mixins.py:214
#, fuzzy
#| msgid "Could not find specified data from Elasticsearch!"
msgid "Couldn't get context from Elasticsearch!"
msgstr "Andmeid ei leitud!"

#: core/mixins.py:293
msgid "Couldn't store ChatGPT results!"
msgstr ""
"Ei suutnud salvestada ChatGPT tulemusi! Palun teavitage administraatorit!"

#: core/models.py:77
#, python-format
msgid "Unknown dataset names: [%(bad_dataset_names)s]"
msgstr "Võõrad andmestiku nimed: [%(bad_dataset_names)s]"

#: core/utilities.py:14
msgid "Minimum year must be lesser than currently running year!"
msgstr "Miinimum aasta piirang peab olema väiksem kui praegune aasta!"

#: core/utilities.py:17
msgid "Minimum year must be lesser than the maximum year restriction!"
msgstr "Miinimum aasta piirang ei tohi olla suurem kui maksimum aasta oma!"

#: document_search/tasks.py:165
msgid "Unknown error, couldn't handle response from ChatGPT!"
msgstr "Teadmata viga ChatGPT ühenduses! Palun teavitage administraatorit!"

#: document_search/views.py:153 text_search/views.py:67
msgid "Deleted chosen objects!"
msgstr "Kustutatud!"

#: user_profile/permissions.py:29
msgid "Your account is not a manager!"
msgstr "Teie kasutaja pole administraator!"

#: user_profile/permissions.py:40
#, fuzzy
#| msgid "Your account has not been accepted by a moderator!"
msgid "Your account is not enabled! Contact an administrator!"
msgstr "Teie kasutaja on välja lülitatud, teavitage administraatorit!"

#: user_profile/permissions.py:51
msgid "You have reached your monetary limit!"
msgstr "Ületasite oma kasutuslimiidi!"

#: user_profile/serializers.py:17
msgid "The given value is not an email."
msgstr "Antud email ei vasta emaili formaadile!"

#: user_profile/serializers.py:24
msgid "The given username is already in use."
msgstr "Antud kasutajanimi on juba kasutuses!"

#: user_profile/serializers.py:31
msgid "The given email is already in use."
msgstr "Antud email on juba kasutuses!"

#: user_profile/serializers.py:124
msgid "The given token is invalid."
msgstr "Turvakontroll ebaõnnestus!"

#: user_profile/serializers.py:129
msgid "Your password reset has expired!"
msgstr "Teie parooli muutmise periood on aegunud!"

#: user_profile/serializers.py:145
msgid "User with that email does not exist!"
msgstr "Sellise emailiga kasutaja ei eksisteeri!"

#: user_profile/views.py:49
msgid "User and password do not match!"
msgstr "Kasutajanimi ja/või parool on vale!"

#: user_profile/views.py:159
msgid "You can not change your own status!"
msgstr "Omaenda admin staatust ei saa muuta!"

#: user_profile/views.py:245
msgid "Sending email failed."
msgstr "Emaili saatmine ebaõnnestus!"

#: user_profile/views.py:266
msgid "Your password has been reset!"
msgstr "Teie parool on muudetud!"

#~ msgid "Can not set acceptance for an already accepted user."
#~ msgstr "Aksepteeritud kasutaja staatust ei saa muuta!"
